druntime/src/core/sys/freebsd/ifaddrs.d

```diff
@@ -0,0 +1,41 @@
+//Written in the D programming language
+
+/++
+    D header file for FreeBSD's ifaddrs.h.
+
+    Copyright: Copyright 2023
+    License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+    Authors:   $(HTTP jmdavisprog.com, Jonathan M Davis)
+ +/
+module core.sys.freebsd.ifaddrs;
+
+import core.sys.posix.sys.socket : sockaddr;
```

Comment:
this should be below the `version (FreeBSD):`, the selective import causes failures on non-posix systems (i.e. Windows)

Reply:
fixed